### DISCUSSION TITLE
Compatibility with Heatzy Pilote Pro ?

### DISCUSSION THREAD

**Discussion by JonasMht** [OlivierZal/com.heatzy#328] [Q&A]
Hello,
I was wondering if this Heatzy integration would work with the Pilote Pro ?

I would like to equip my home with such a system but am waiting to find a homey compatible product

**Comment by OlivierZal** [comment]
Hi @JonasMht, in theory it should work because my app integrated support based on what can be found in the Heatzy documentation, but since I don't have this device I cannot confirm it works...

**Reply by JonasMht** [reply]
Thanks, I'll keep you posted on this if I get the system.
Cheers
